src/System.Net.Security/tests/FunctionalTests/NegotiateStreamStreamToStreamTest.cs

```diff
@@ -90,6 +90,8 @@ public abstract class NegotiateStreamStreamToStreamTest
             {
                 Assert.False(client.IsAuthenticated);
                 Assert.False(server.IsAuthenticated);
+                Assert.Equal(false, client.IsMutuallyAuthenticated);
+                Assert.Equal(false, server.IsMutuallyAuthenticated);
```

Comment:
These 'Assert.Equal(false,' should be 'Assert.False('

Reply:
I don't understand why this test is named "_Success".  Nothing seems successful if all those properties are false.

What is this testing exactly? I think it is testing a failure condition where the stream ends up un-authenticated. So, perhaps the test should be named like:

"NegotiateStream_StreamToStream_Authentication_UseInvalidTargetName_UnauthenticatedStream"

Reply:
> Nothing seems successful if all those properties are false.
> What is this testing exactly?

This test is asserting some properties values in NegotiateStream before and after authentication, to verify that the authentication with TargetName can succeed. This block of `Assert.False()` happens before calling into AuthenticateAsClientAsync/AuthenticateAsServerAsync, so those values are expected to be false. Therefore, the name to describe the purpose of this test method is correct.

Will fix the `Assert.Equal(false` problem.